README.md

```diff
@@ -29,21 +121,23 @@ As the maintainer of this project, please make a few updates:
 - `test_data`/`test_size` is not supported at the same time as `cv_split_column_names`
 - `test_data`/`test_size` is not supported at the same time as `cv_splits_indices`
 - Test runs are not supported for AutoML runs which have streaming enabled.
```

Comment:
I'm not sure I was aware of this: "Test runs are not supported for AutoML runs which have streaming enabled".
Why is that? Why models/algos (Based on NimbusML I guess) using data streaming do not support it?

Also, users don't really "Enable streaming", there's no such parameter in AutoMLConfig class, so if this is happening we need to be more specific/explicit and list the models/algos not supported by Model-Test. Right?

Reply:
Updated the docs to mention how the workspaces and resource groups can be made more secure.

Reply:
Updated the wording to only mention the model selection criteria which are used to determine if a run will be a streaming run.